activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -47,9 +47,12 @@ class RedisCacheStore < Store
         reconnect_attempts: 0,
       }
 
-      DEFAULT_ERROR_HANDLER = -> (method:, returning:, exception:) {
-        logger.error { "RedisCacheStore: #{method} failed, returned #{returning.inspect}: #{e.class}: #{e.message}" } if logger
-      }
+      DEFAULT_ERROR_HANDLER = -> (method:, returning:, exception:) do
+        if logger
+          message = "#{name}: #{method} failed, returned #{returning.inspect}: #{exception.class}: #{exception.message}"
+          logger.error { message }
```

Comment:
The message string is within the block so it needn't be generated unless the logger level is `error` or greater.

Reply:
Good call! I'll update it!